common/src/main/java/org/wildfly/httpclient/common/HttpTargetContext.java

```diff
@@ -37,33 +55,10 @@
 import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.zip.GZIPInputStream;
-import javax.net.ssl.SSLContext;
-
-import org.jboss.marshalling.InputStreamByteInput;
-import org.jboss.marshalling.Marshaller;
-import org.jboss.marshalling.MarshallingConfiguration;
-import org.jboss.marshalling.SimpleClassResolver;
-import org.jboss.marshalling.Unmarshaller;
-import org.wildfly.security.auth.client.AuthenticationConfiguration;
-import org.wildfly.security.auth.client.AuthenticationContext;
-import org.wildfly.security.auth.client.AuthenticationContextConfigurationClient;
-import org.xnio.ChannelListener;
-import org.xnio.ChannelListeners;
-import org.xnio.IoUtils;
-import org.xnio.channels.StreamSourceChannel;
-import io.undertow.client.ClientCallback;
-import io.undertow.client.ClientExchange;
-import io.undertow.client.ClientRequest;
-import io.undertow.client.ClientResponse;
-import io.undertow.server.handlers.Cookie;
-import io.undertow.util.AbstractAttachable;
-import io.undertow.util.Cookies;
-import io.undertow.util.HeaderValues;
-import io.undertow.util.Headers;
-import io.undertow.util.Methods;
-import io.undertow.util.StatusCodes;
```

Comment:
@bstansberry I checked this and I can't find any codestyle for those projects. I started using the codestyle that WildFly uses at least so I can have a standard to follow. However, I applied it and hte differences in the imports order continued. the same as it is in this PR.

Reply:
@fl4via Where is the thing you applied?

If you mean what's in https://github.com/wildfly/wildfly-checkstyle-config, we don't enforce import ordering via checkstyle, as it would cause a ton of failures. It's just a general convention that we've tried to enforce via code review for a long time. Mostly we comment on new files if we notice, and more often we comment on PRs that rearrange the order.

Reply:
I've created https://issues.redhat.com/browse/WEJBHTTP-84 @bstansberry .
I didn't assign it a fix version because next release doesn't exist in WEJBHTTP JIRA project yet.
@fl4via could you please create 1.2.0.Final version in WEJBHTTP JIRA please?